packages/common/src/surveys/_internal/SurveyUiSchemaSettings.ts

```diff
@@ -0,0 +1,53 @@
+import { IArcGISContext } from "../../ArcGISContext";
+import { EntityEditorOptions } from "../../core/schemas/internal/EditorOptions";
+import { IUiSchema, UiSchemaRuleEffects } from "../../core/schemas/types";
+
+/**
+ * @private
+ * settings uiSchema for Hub Survey - this
+ * defines how the schema properties should be
+ * rendered in the Survey settings experience
+ */
+export const buildUiSchema = async (
+  i18nScope: string,
+  options: EntityEditorOptions,
+  context: IArcGISContext
+): Promise<IUiSchema> => {
+  return {
+    type: "Layout",
+    elements: [
+      {
+        type: "Section",
+        rule: {
```

Comment:
After looking more closely at [figma](https://www.figma.com/file/1bvSoMV4ygNqHGXmkIBgJd/Survey-workspace?type=design&node-id=0-1&mode=design&t=hAqQjRwSraKjf8OR-0), we don't want to conditionally render this section, we instead want to _always_ render this section, but _disable_ the tile-select when no map question is configured.  We also need to conditionally render a calcite notice when no map question is configured. This will involve:

1. Move this entire `rule` to the 0-index child element of this section that defines the tile-select control
2. Change the `effect` to `UiSchemaRuleEffects.DISABLE`
3. Define a `messages` array that is empty when `options.hasMapQuestion` is truthy, else an array containing a single message config object when `options.hasMapQuestion` is falsey.
4. Expanding the IUiSchemaMessage in `packages/common/src/core/schemas/types.ts` to define optional `title` and `titleKey` properties (IUiSchemaMessage currently only support defining the calcite-notice `message` slot value which is mapped from `label` or `labelKey` props on this interface). This will also require us to do a little bit more work in opendata-ui to support, adding specifics to your other PR.

```ts
// packages/common/src/surveys/_internal/SurveyUiSchemaSettings.ts

export const buildUiSchema = async (
  i18nScope: string,
  options: EntityEditorOptions,
  context: IArcGISContext
): Promise<IUiSchema> => {
  return {
    type: "Layout",
    elements: [
      {
        type: "Section",
        labelKey: `${i18nScope}.sections.settings.label`,
        elements: [
          {
            labelKey: `${i18nScope}.fields.displayMap.label`,
            scope: "/properties/displayMap",
            type: "Control",
            rule: {
              effect: UiSchemaRuleEffects.DISABLE,
              condition: {
                scope: "/properties/hasMapQuestion",
                schema: { const: false },
              },
            },
            options: {
              control: "hub-field-input-tile-select",
              type: "radio",
              labels: [
                `{{${i18nScope}.fields.displayMap.enabled.label:translate}}`,
                `{{${i18nScope}.fields.displayMap.disabled.label:translate}}`,
              ],
              descriptions: [
                `{{${i18nScope}.fields.displayMap.enabled.description:translate}}`,
                `{{${i18nScope}.fields.displayMap.disabled.description:translate}}`,
              ],
              icons: ["sidecar", "form-elements"],
              layout: "horizontal",
              messages: (options as IHubSurvey).hasMapQuestion
                ? []
                : [
                    {
                      type: UiSchemaMessageTypes.custom,
                      display: "notice",
                      keyword: "mapQuestion",
                      titleKey: `${i18nScope}.fields.displayMap.notice.title`,
                      labelKey: `${i18nScope}.fields.displayMap.notice.message`,
                      allowShowBeforeInteract: true,
                      alwaysShow: true,
                    },
                  ],
            },
          },
        ],
      },
    ],
  };
};
```

```ts
// packages/common/src/core/schemas/types.ts

export interface IUiSchemaMessage {
  type: UiSchemaMessageTypes;
  display?: "message" | "notice";
  keyword?: string;
  title?: string;
  titleKey?: string;
  label?: string;
  labelKey?: string;
  icon?: boolean | string;
  kind?: "brand" | "danger" | "info" | "success" | "warning";
  hidden?: boolean;
  condition?: IUiSchemaCondition;
  allowShowBeforeInteract?: boolean;
  alwaysShow?: boolean;
}
```